apps/calendar/src/components/timeGrid/times.tsx

```diff
@@ -147,14 +147,7 @@ export function Times({
           </div>
         );
       })}
-      {showCurrentTime ? (
-        <CurrentTimeLabel
-          unit={unit}
-          time={currentTime}
-          top={top}
-          dateDifference={dateDifference}
-        />
-      ) : null}
+      {showCurrentTime ? <CurrentTimeLabel unit={unit} time={currentTime} top={top} /> : null}
```

Comment:
```suggestion
      {showCurrentTime && <CurrentTimeLabel unit={unit} time={currentTime} top={top} />}
```

삼항연산자 말고 `&&`를 써도 되지 않을까요?

Reply:
그러게요. 일단 `&&` 으로 고칠게요. 참고로 컴포넌트 렌더링에 `&&` 을 사용하는 경우는 실수할 여지가 있으니 가급적이면 확실하게 불린값으로 구분되는 경우만 사용하는걸 권장하고 싶습니다.

https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx